lumicks/pylake/population/dwelltime.py

```diff
@@ -967,7 +1032,8 @@ def _exponential_mle_optimize(
 
     if initial_guess is None:
         initial_guess_amplitudes = np.ones(n_components) / n_components
-        initial_guess_lifetimes = np.mean(t) * np.arange(1, n_components + 1)
+        fractions = np.arange(1, n_components + 1)
+        initial_guess_lifetimes = np.mean(t) * n_components * fractions / np.sum(fractions)
```

Comment:
how did you come up with this? is it some standard way of making initial guesses?

Reply:
Not really, but I figured that the weighted average of the means should be the total mean to make sure it doesn't get really large when the number of components is large.